<?xml version="1.0"?>
<launch>

  <group ns="main_robot">
    <node pkg="urg_node" name="lidar_node" type="urg_node">
      <param name="foo" value="192.168.0.10"/>
    </node>
    <node pkg="eurobot_decisiin_maker" name="particle_filter_node" type="particle_filter_node.py"/>
    <node pkg="eurobot_decision_maker" name="stm_node" type="stm_node.py"/>
    <node pkg="eurobot_decision_maker" name="track_regulator" type="track_regulator.py"/>
  </group>

</launch>
